modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/field/expression/handler/MembershipsUserFieldExpressionHandlerRegistrator.java

```diff
@@ -25,23 +37,105 @@
 @Component(service = {})
 public class MembershipsUserFieldExpressionHandlerRegistrator {
 
-	@Activate
-	protected void activate(BundleContext bundleContext) {
-		if (!FeatureFlagManagerUtil.isEnabled("LPS-180198")) {
-			return;
+	public class MembershipsUserFieldExpressionHandler
```

Comment:
Hey @regiCesar21, keep it as a non-internal class. It will reduce the line changes and still cover our goal. Thx